src/components/ListLink/index.js

```diff
@@ -63,12 +63,22 @@ const ListLink = (props) => {
           }
 
           &:focus {
-            outline: 0.2rem solid ${theme.colors.accent.primaryLight};
-            outline-offset: 0.4rem;
+            outline: 0.2rem solid ${theme.colors.state.focusOutline};
 
             span::after {
               border-color: transparent;
             }
+
+            ${iconName &&
+              css`
+                outline: none;
+
+                .icon::before {
```

Comment:
Hmmm, where does this class name come from?

Did we give icons in Nautilus this class name? I thought it was more verbose… this seems a very un-scoped class name to be exposing! We should fix that in Nautilus if so.

Reply:
*cough cough*

https://github.com/octopusthink/getmicdrop.com/blob/2d2aacc0997acb862a663db50c3ccd83176150f4/src/components/ListLink/index.js#L116

Reply:
Nautilus gives the icon an unpredictable class name, so we assign one to the icon here for easier CSS-ing from parent components. There are a few places where we use this approach.

Reply:
Works for me!